src/platforms/rust/index.mdx

```diff
@@ -45,6 +45,43 @@ let _guard = sentry::init(("___PUBLIC_DSN___", sentry::ClientOptions {
 
 **Important:** Note your DSN. The DSN (Data Source Name) tells the SDK where to send events. If you forget it, view Settings -> Projects -> Client Keys (DSN) in sentry.io.
 
+### Async main function
+
+The Sentry client must be initialized before any async runtime is started or threads are spawned. Unfortunately, this makes it impossible to use macros such as `#[tokio::main]` or `#[actix_web::main]`, because they start the runtime first. Thus, instead of
```

Comment:
```suggestion
The Sentry client must be initialized before any async runtime is started, otherwise threads will be spawned. This means you'll have to avoid using macros such as `#[tokio::main]` or `#[actix_web::main]`, because they start the runtime first. So rather than doing this:
```

Reply:
Ah, here my formulation was clearly ambiguous. I meant "The Sentry client must be initialized before (any async runtime is started or threads are spawned)".

How does 
> The Sentry client must be initialized before starting an async runtime or spawning threads.

sound?